exercises/03_reverse/reverse.go

```diff
@@ -1,4 +1,9 @@
 package main
 
-func main() {
+func reverse(n int) int {
+	result := 0
```

Comment:
A variable initialized with a default value should be declared as `var result int`.